src/tests/unittests/asn1.c

```diff
@@ -439,6 +439,124 @@ static void torture_asn1_decode_entry_bit_string_ni(void **state)
 	assert_memory_equal(bit_string + 1, result, resultlen/8);
 }
 
+static void torture_asn1_put_tag_short(void **state)
+{
```

Comment:
clang-format suggested the following change:


```
-static void torture_asn1_put_tag_short(void **state)
-{
+static void torture_asn1_put_tag_short(void **state) {
```

<!-- identifier: clang-format -->
<!-- File: src/tests/unittests/asn1.c -->